pyproject.toml

```diff
@@ -25,7 +25,7 @@ pydantic = "^1.6"
 rich = "^9.5"
 jsonpointer = "^2.1"
 jmespath = "^0.10"
-ansible = { version = "^2.10.0", optional = true }
+ansible = { version = "^4.2", optional = true }
 ansible-base = { version = "^2.10.0", optional = true }
```

Comment:
Does this need to be bumped as well?

Reply:
It doesn't need to be bumped per resolving the CVE.